Sources/SwiftDocC/Model/Rendering/Tutorial/References/DownloadReference.swift

```diff
@@ -39,21 +53,14 @@ public struct DownloadReference: RenderReference, URLReference {
     public init(identifier: RenderReferenceIdentifier, renderURL url: URL, sha512Checksum: String?) {
```

Comment:
It looks like we also need to create a new initializer and deprecate the prior non-optional one: https://github.com/apple/swift-docc/commit/c9a5c5e4baa4d49a0b8deba8eee8a7c880091059#diff-475e1b586505aa67d65b7495b5d7f45a02b5477adb90fa3395b4f915c3506bddL39

Reply:
Done; i don't think this is a particularly breaking change, since a non-optional String should be able to be converted to an optional String without change, but it's still worth keeping around.